### PR TITLE
Enhance listener implementation

### PR DESCRIPTION
**Pull request by khlifi411**
**Changes proposed in this PR**

Set `kyma-name` label on generic event object to identify Kyma CR.

### CODE REVIEW

File: kcp/pkg/listener/skr_events_listener.go [hunk 1]
```diff
@@ -44,15 +44,17 @@ func (l *SKREventListener) Start(ctx context.Context) error {
...
+	router.HandleFunc(listenerPattern, l.handleSKREvent())
 
 	//start web server
 	server := &http.Server{Addr: l.addr, Handler: router}
 	go func() {
-		l.logger.Info("SKR events listener is starting up...")
+		l.logger.WithValues(
+			"Addr", l.addr,
+			"ApiPath", listenerPattern,
+		).Info("Listener is starting up...")
```
Comment (adityabhatia) [inline kcp/pkg/listener/skr_events_listener.go:57]: please add port here
```suggestion
		).Info("Listener is starting up on port xxxx")
```

**Review by adityabhatia — DISMISSED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Review by adityabhatia — APPROVED** [review]

**Pull request merged**